### PR TITLE
Sanbox branch

### PR DESCRIPTION
**Pull request by lartist**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop / 8.1.x / 8.0.x / 1.7.8.x
| Description?      | Please be specific when describing the PR. <br> Every detail helps: versions, browser/server configuration, specific module/theme, etc. Feel free to add more information below this table.
| Type?             | bug fix / improvement / new feature / refacto
| Category?         | FO / BO / CO / IN / WS / TE / LO / ME / PM
| BC breaks?        | yes / no
| Deprecations?     | yes / no
| How to test?      | Indicate how to verify that this change works as expected.
| Fixed ticket?     | Fixes #{issue number here}, Fixes #{another issue number here}
| Related PRs       | If theme, autoupgrade or other module change is needed to make this change work, provide a link to related PRs here.
| Sponsor company   | Your company or customer's name goes here (if applicable).

### CODE REVIEW

**Review by M0rgan01 — CHANGES_REQUESTED** [review]
May the force be with you

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+Peace be upon you
```
Comment (M0rgan01) [inline README.md:1]:
```suggestion
May the force be with you
```

**Review by M0rgan01 — CHANGES_REQUESTED** [review]

**Review by 0x346e3730 — CHANGES_REQUESTED** [review]
NO

**Review by FabienPapet — CHANGES_REQUESTED** [review]
https://media0.giphy.com/media/vyTnNTrs3wqQ0UIvwE/giphy-downsized.gif?cid=6104955eifeblja5hioivk88x8nhfuiyubdkv2q932ynqgpm&ep=v1_gifs_translate&rid=giphy-downsized.gif&ct=g

**Review by nicosomb — APPROVED** [review]
LGTM, go merge.

**Pull request closed without merging**